src/components/help/HelpContainer.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { ReactComponent as ExitSVG } from 'media/icons/times-light.svg';
+import { ReactComponent as QuestionCircleSVG } from 'media/icons/question-circle-solid.svg';
+import HelpPanel from './HelpPanel';
+
+const styles = (theme) => ({
+  panelContainer: {
+    color: '#333F52',
+    fontFamily: theme.typography.fontFamily,
+    right: '0',
+    'max-width': '815px',
+    height: 'calc(100% - 64px)', // 64px = height of top nav bar
```

Comment:
Is that a value we can pull from somewhere

Reply:
I added a constant in theme to store this.. but I'm not sure that makes sense. I don't see the direct height set anywhere. Thoughts?